docs/src/main/tut/json.md

```diff
@@ -243,7 +243,7 @@ Finally, we post `User("Alice")` to our Hello service and expect
 `Hello("Alice")` back:
 
 ```tut:book
-helloClient("Alice").run
+//helloClient("Alice").unsafeRun
```

Comment:
Same probably here as in client.md. Tut compile never completes probably because the Task never completes. I'm probably missing something stupid, but I haven't figured it out, yet.